api/multiformat/utils.go

```diff
@@ -18,7 +18,7 @@ const (
 	secp256k1KeyType   = 0xe7
 	bls12p381g1KeyType = 0xea
 	bls12p381g2KeyType = 0xeb
-	legacyKeyLength    = 132
+	LegacyKeyLength    = 132
```

Comment:
Made this public to perform a validation needed in the new implementation :eyes: